 * Tip: There are .env files present. Do "pip install python-dotenv" to use them.
 * Serving Flask app "flaskr" (lazy loading)
 * Environment: development
 * Debug mode: on
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Tip: There are .env files present. Do "pip install python-dotenv" to use them.
 * Debugger is active!
 * Debugger PIN: 138-361-213
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
192.168.31.18 - - [02/Mar/2021 16:44:35] "GET / HTTP/1.1" 302 -
192.168.31.18 - - [02/Mar/2021 16:44:38] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 16:44:38] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 16:44:38] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 16:44:38] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 16:44:53] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 16:44:53] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 16:44:53] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 16:44:53] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 16:44:54] "GET /fund_grid HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 16:44:54] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 16:44:55] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 16:44:55] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 16:44:55] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 17:03:00] "GET /fund_grid HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 17:03:00] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 17:03:01] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 17:03:01] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 17:03:01] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 17:03:04] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 17:03:04] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 17:03:04] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 17:03:04] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 17:03:52] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 19:52:57] "GET /fund_grid HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 19:52:57] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 21:17:33] "GET / HTTP/1.1" 302 -
192.168.31.18 - - [02/Mar/2021 21:17:35] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 21:17:36] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 21:17:36] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 21:17:36] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 21:17:36] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 21:17:39] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 21:17:39] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 21:17:39] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 21:17:47] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 21:17:47] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 21:17:47] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 21:17:47] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 21:17:47] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 22:09:55] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [02/Mar/2021 22:09:55] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 22:09:55] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 22:09:55] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [02/Mar/2021 22:09:55] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.98 - - [02/Mar/2021 22:28:46] "GET / HTTP/1.1" 302 -
192.168.31.98 - - [02/Mar/2021 22:28:48] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [02/Mar/2021 22:28:49] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.98 - - [02/Mar/2021 22:28:50] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 200 -
192.168.31.98 - - [02/Mar/2021 22:28:50] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.98 - - [02/Mar/2021 22:28:50] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.98 - - [02/Mar/2021 22:28:50] "GET /static/favicon.ico HTTP/1.1" 200 -
192.168.31.18 - - [03/Mar/2021 09:07:15] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [03/Mar/2021 09:07:16] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 09:07:16] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 09:07:16] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 09:07:16] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 09:49:04] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [03/Mar/2021 09:49:04] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 09:49:04] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 09:49:04] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 09:49:04] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 09:49:05] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.18 - - [03/Mar/2021 09:49:06] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 09:49:06] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 304 -
192.168.31.98 - - [03/Mar/2021 19:37:01] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [03/Mar/2021 19:37:01] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.98 - - [03/Mar/2021 19:37:01] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.98 - - [03/Mar/2021 19:37:01] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.98 - - [03/Mar/2021 19:40:02] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [03/Mar/2021 19:40:02] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.98 - - [03/Mar/2021 19:40:02] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.98 - - [03/Mar/2021 19:40:02] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.98 - - [03/Mar/2021 19:40:15] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
192.168.31.98 - - [03/Mar/2021 19:40:15] "GET /fund?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
192.168.31.98 - - [03/Mar/2021 19:40:15] "GET /fund?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
192.168.31.98 - - [03/Mar/2021 19:40:15] "GET /fund?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
192.168.31.98 - - [03/Mar/2021 19:40:16] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.98 - - [03/Mar/2021 19:40:16] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.18 - - [03/Mar/2021 20:21:21] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [03/Mar/2021 20:21:21] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 20:21:21] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 20:21:21] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [03/Mar/2021 20:21:21] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 09:00:27] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 09:00:27] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 09:00:27] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 09:00:27] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 09:00:28] "GET /static/pics/weather_forecast/%E9%9B%B7%E9%98%B5%E9%9B%A8.png HTTP/1.1" 404 -
192.168.31.18 - - [04/Mar/2021 09:23:31] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 09:23:32] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 09:23:32] "GET /static/pics/weather_forecast/%E9%9B%B7%E9%98%B5%E9%9B%A8.png HTTP/1.1" 404 -
192.168.31.18 - - [04/Mar/2021 09:23:32] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 09:23:32] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 16:38:02] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 16:38:03] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 404 -
192.168.31.18 - - [04/Mar/2021 16:38:03] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 16:38:03] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 16:38:03] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 16:40:12] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 17:11:31] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 17:11:33] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 17:11:33] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 17:11:33] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 17:11:33] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 404 -
192.168.31.98 - - [04/Mar/2021 17:25:05] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [04/Mar/2021 17:25:05] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.98 - - [04/Mar/2021 17:25:06] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.98 - - [04/Mar/2021 17:25:06] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.98 - - [04/Mar/2021 17:25:06] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 404 -
192.168.31.98 - - [04/Mar/2021 17:25:06] "GET /static/favicon.ico HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:49:43] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:49:43] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:49:43] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 404 -
192.168.31.18 - - [04/Mar/2021 18:49:43] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:49:43] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:14] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:15] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 18:58:15] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:15] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 18:58:15] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 18:58:34] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
192.168.31.18 - - [04/Mar/2021 18:58:35] "GET /fund?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:35] "GET /fund?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:35] "GET /fund?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:35] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:35] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:37] "GET /fund_grid HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:38] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 18:58:39] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:39] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 18:58:39] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 18:58:42] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [04/Mar/2021 18:58:43] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 18:58:43] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 18:58:43] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.18 - - [04/Mar/2021 18:58:43] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.98 - - [04/Mar/2021 19:12:52] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [04/Mar/2021 19:12:52] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.98 - - [04/Mar/2021 19:12:52] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.98 - - [04/Mar/2021 19:12:52] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.98 - - [04/Mar/2021 19:12:52] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.98 - - [04/Mar/2021 20:20:49] "GET /index HTTP/1.1" 200 -
 * Tip: There are .env files present. Do "pip install python-dotenv" to use them.
 * Serving Flask app "flaskr" (lazy loading)
 * Environment: development
 * Debug mode: on
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Tip: There are .env files present. Do "pip install python-dotenv" to use them.
 * Debugger is active!
 * Debugger PIN: 138-361-213
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
192.168.31.18 - - [05/Mar/2021 20:56:59] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [05/Mar/2021 20:57:01] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [05/Mar/2021 20:57:01] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [05/Mar/2021 20:57:01] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [05/Mar/2021 20:57:01] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [05/Mar/2021 20:57:01] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.18 - - [06/Mar/2021 09:58:48] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [06/Mar/2021 09:58:49] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [06/Mar/2021 09:58:49] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [06/Mar/2021 09:58:49] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.18 - - [06/Mar/2021 09:58:49] "GET /static/pics/weather_forecast/%E9%9B%B7%E9%98%B5%E9%9B%A8.png HTTP/1.1" 404 -
192.168.31.18 - - [06/Mar/2021 09:58:49] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.18 - - [06/Mar/2021 09:59:10] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [06/Mar/2021 10:06:34] "GET /index HTTP/1.1" 200 -
192.168.31.18 - - [06/Mar/2021 10:06:34] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.18 - - [06/Mar/2021 10:06:34] "GET /static/pics/weather_forecast/%E9%9B%B7%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.18 - - [06/Mar/2021 10:06:34] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.18 - - [06/Mar/2021 10:06:34] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.18 - - [06/Mar/2021 10:06:34] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.18 - - [06/Mar/2021 10:06:56] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
192.168.31.18 - - [06/Mar/2021 10:06:57] "GET /fund?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
192.168.31.18 - - [06/Mar/2021 10:06:57] "GET /fund?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
192.168.31.18 - - [06/Mar/2021 10:06:57] "GET /fund?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
192.168.31.18 - - [06/Mar/2021 10:06:57] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/home/pi/EpsAvlc_index/flaskr/blueprints/index.py', reloading
223.73.111.148
223.73.111.148
223.73.111.148
223.73.111.148
223.73.111.148
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
 * Restarting with stat
 * Tip: There are .env files present. Do "pip install python-dotenv" to use them.
 * Debugger is active!
 * Debugger PIN: 138-361-213
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
192.168.31.164 - - [07/Mar/2021 15:03:49] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [07/Mar/2021 15:03:50] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [07/Mar/2021 15:03:50] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [07/Mar/2021 15:03:50] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.164 - - [07/Mar/2021 15:03:51] "GET /static/favicon.ico HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 08:46:38] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 08:55:08] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 08:55:08] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 08:55:08] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 08:55:08] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 08:55:09] "GET /static/favicon.ico HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 09:14:36] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 09:14:44] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 09:14:44] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 09:14:44] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.98 - - [11/Mar/2021 09:14:44] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.98 - - [11/Mar/2021 09:14:47] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.98 - - [11/Mar/2021 09:14:47] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.98 - - [11/Mar/2021 09:14:47] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 10:42:16] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 10:42:17] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [12/Mar/2021 10:42:17] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [12/Mar/2021 11:01:25] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
192.168.31.164 - - [12/Mar/2021 11:01:25] "GET /fund?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:25] "GET /fund?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:25] "GET /fund?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:25] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:25] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:28] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
192.168.31.164 - - [12/Mar/2021 11:01:29] "GET /fund?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:29] "GET /fund?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:29] "GET /fund?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:29] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:39] "GET /fund_grid HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:40] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [12/Mar/2021 11:01:41] "GET /fund_grid HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:41] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [12/Mar/2021 11:01:41] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:01:41] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:06:54] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
192.168.31.164 - - [12/Mar/2021 11:06:55] "GET /fund?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:06:55] "GET /fund?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:06:55] "GET /fund?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:06:55] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
 * Detected change in '/home/pi/EpsAvlc_index/flaskr/__init__.py', reloading
223.73.111.148
223.73.111.148
223.73.111.148
223.73.111.148
223.73.111.148
223.73.111.148
 * Restarting with stat
 * Tip: There are .env files present. Do "pip install python-dotenv" to use them.
 * Debugger is active!
 * Debugger PIN: 138-361-213
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
192.168.31.164 - - [12/Mar/2021 11:21:17] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:21:17] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [12/Mar/2021 11:21:17] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 200 -
 * Tip: There are .env files present. Do "pip install python-dotenv" to use them.
 * Serving Flask app "flaskr" (lazy loading)
 * Environment: development
 * Debug mode: on
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Tip: There are .env files present. Do "pip install python-dotenv" to use them.
 * Debugger is active!
 * Debugger PIN: 138-361-213
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
192.168.31.164 - - [17/Mar/2021 11:11:06] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
192.168.31.164 - - [17/Mar/2021 11:11:07] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 11:11:07] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 11:11:07] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 11:11:11] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 11:11:12] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 11:11:12] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 11:11:12] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 11:11:12] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 11:11:29] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 11:11:29] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 11:11:29] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 11:11:33] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 11:11:33] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 11:11:33] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 11:11:33] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 11:11:33] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 14:53:21] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 14:53:21] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 14:53:21] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 14:53:21] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 14:53:21] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 15:32:01] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 15:32:02] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 15:32:02] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 15:32:02] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 15:32:02] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 15:49:38] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
192.168.31.164 - - [17/Mar/2021 15:49:38] "GET /fund?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 15:49:38] "GET /fund?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 15:49:38] "GET /fund?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 15:49:38] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 19:40:13] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 19:40:14] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 19:40:14] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 19:40:14] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 19:40:14] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 21:02:10] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [17/Mar/2021 21:02:10] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 21:02:11] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 21:02:11] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 21:02:11] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [17/Mar/2021 21:02:11] "GET /static/favicon.ico HTTP/1.1" 200 -
192.168.31.164 - - [18/Mar/2021 15:13:04] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [18/Mar/2021 15:13:18] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [18/Mar/2021 15:13:18] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.164 - - [18/Mar/2021 15:13:18] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [18/Mar/2021 15:13:18] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [18/Mar/2021 21:23:58] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [18/Mar/2021 21:23:59] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [18/Mar/2021 21:23:59] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.164 - - [18/Mar/2021 21:23:59] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [18/Mar/2021 21:23:59] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 09:09:24] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 09:09:24] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 09:09:24] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 09:09:24] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 09:09:25] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 10:19:05] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 10:19:06] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 10:19:06] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 10:19:06] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 10:19:06] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 14:59:35] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 14:59:43] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 14:59:43] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 14:59:43] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 14:59:43] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 15:39:21] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 15:39:27] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 15:39:28] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 15:39:28] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 15:39:28] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 20:27:23] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 20:27:23] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 20:27:23] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 20:27:23] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 22:14:44] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [19/Mar/2021 22:14:44] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 22:14:44] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [19/Mar/2021 22:14:44] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [20/Mar/2021 10:01:25] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [20/Mar/2021 10:01:26] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [20/Mar/2021 10:01:26] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [20/Mar/2021 10:01:26] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.98 - - [20/Mar/2021 10:24:47] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [20/Mar/2021 10:24:49] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.98 - - [20/Mar/2021 10:24:49] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.98 - - [20/Mar/2021 10:24:49] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.98 - - [20/Mar/2021 10:24:50] "GET /static/favicon.ico HTTP/1.1" 200 -
192.168.31.164 - - [20/Mar/2021 18:58:09] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [20/Mar/2021 19:06:26] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 200 -
192.168.31.164 - - [20/Mar/2021 19:06:26] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [20/Mar/2021 19:06:26] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [20/Mar/2021 20:59:53] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [20/Mar/2021 20:59:54] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [20/Mar/2021 20:59:54] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 09:14:48] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 09:14:48] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [21/Mar/2021 09:14:48] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [21/Mar/2021 09:14:48] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [21/Mar/2021 09:14:48] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [21/Mar/2021 18:01:29] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 18:01:30] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 18:01:30] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 18:01:30] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 18:01:30] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 18:29:26] "GET /onmyoji HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 18:29:26] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 18:29:26] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 20:41:14] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [21/Mar/2021 20:41:15] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [21/Mar/2021 20:41:15] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [21/Mar/2021 20:41:15] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [22/Mar/2021 11:16:45] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [22/Mar/2021 11:16:45] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.164 - - [22/Mar/2021 11:16:45] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [22/Mar/2021 11:16:45] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [22/Mar/2021 20:50:32] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [22/Mar/2021 20:50:32] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [22/Mar/2021 20:50:32] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
192.168.31.164 - - [22/Mar/2021 20:50:32] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [22/Mar/2021 20:51:44] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [22/Mar/2021 20:51:44] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [22/Mar/2021 20:51:44] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [22/Mar/2021 20:51:44] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 16:51:00] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 16:51:01] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [24/Mar/2021 16:51:01] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [24/Mar/2021 16:51:05] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
192.168.31.164 - - [24/Mar/2021 16:51:06] "GET /fund?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 16:51:06] "GET /fund?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 16:51:06] "GET /fund?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 16:51:06] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 17:04:50] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 17:04:50] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [24/Mar/2021 17:04:50] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [24/Mar/2021 17:06:15] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 17:06:15] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [24/Mar/2021 17:06:16] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [24/Mar/2021 19:40:43] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 19:40:44] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 22:34:32] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 22:34:33] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 22:34:33] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 22:35:31] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 22:35:31] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.164 - - [24/Mar/2021 22:35:31] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [25/Mar/2021 09:48:17] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [25/Mar/2021 09:48:17] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [25/Mar/2021 09:48:17] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.164 - - [25/Mar/2021 10:13:35] "GET / HTTP/1.1" 302 -
192.168.31.164 - - [25/Mar/2021 10:13:37] "GET /index HTTP/1.1" 200 -
192.168.31.164 - - [25/Mar/2021 10:13:38] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.164 - - [25/Mar/2021 10:13:38] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [25/Mar/2021 10:23:32] "GET / HTTP/1.1" 302 -
192.168.31.1 - - [25/Mar/2021 10:23:35] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [25/Mar/2021 10:23:35] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.1 - - [25/Mar/2021 10:23:35] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.1 - - [25/Mar/2021 10:23:35] "GET /static/favicon.ico HTTP/1.1" 200 -
192.168.31.1 - - [25/Mar/2021 10:23:57] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [25/Mar/2021 10:23:57] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [25/Mar/2021 10:23:57] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.98 - - [25/Mar/2021 18:15:15] "GET /index HTTP/1.1" 200 -
192.168.31.98 - - [25/Mar/2021 18:15:15] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 200 -
192.168.31.98 - - [25/Mar/2021 18:15:15] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.98 - - [25/Mar/2021 18:15:15] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [25/Mar/2021 19:48:23] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [25/Mar/2021 19:48:24] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.1 - - [25/Mar/2021 19:48:24] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.1 - - [25/Mar/2021 19:48:24] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 200 -
192.168.31.1 - - [25/Mar/2021 20:39:40] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [25/Mar/2021 20:39:41] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [25/Mar/2021 20:39:41] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [25/Mar/2021 20:39:41] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 304 -
192.168.31.1 - - [26/Mar/2021 14:29:53] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [26/Mar/2021 14:29:54] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [26/Mar/2021 14:29:54] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [26/Mar/2021 14:30:00] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
192.168.31.1 - - [26/Mar/2021 14:30:01] "GET /fund?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
192.168.31.1 - - [26/Mar/2021 14:30:01] "GET /fund?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
192.168.31.1 - - [26/Mar/2021 14:30:01] "GET /fund?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
192.168.31.1 - - [26/Mar/2021 14:30:01] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.1 - - [26/Mar/2021 14:30:01] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
192.168.31.1 - - [27/Mar/2021 09:22:57] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [27/Mar/2021 10:12:19] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [27/Mar/2021 10:12:20] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [27/Mar/2021 10:12:20] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [29/Mar/2021 08:46:54] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [29/Mar/2021 08:46:54] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.1 - - [29/Mar/2021 08:46:54] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.1 - - [29/Mar/2021 08:46:54] "GET /static/favicon.ico HTTP/1.1" 200 -
192.168.31.1 - - [29/Mar/2021 14:35:02] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [29/Mar/2021 14:35:02] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [29/Mar/2021 14:35:02] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [29/Mar/2021 21:22:04] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [29/Mar/2021 21:22:04] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.1 - - [29/Mar/2021 21:22:05] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
10.249.78.110 - - [29/Mar/2021 23:25:02] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [31/Mar/2021 20:11:22] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [31/Mar/2021 20:11:22] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [31/Mar/2021 20:11:22] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [02/Apr/2021 09:53:33] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [02/Apr/2021 09:53:33] "GET /static/pics/weather_forecast/%E6%99%B4.png HTTP/1.1" 200 -
192.168.31.1 - - [02/Apr/2021 09:53:33] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [02/Apr/2021 09:53:33] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [02/Apr/2021 17:05:19] "GET /index HTTP/1.1" 200 -
10.249.78.110 - - [06/Apr/2021 10:52:21] "GET /index HTTP/1.1" 200 -
10.249.78.110 - - [06/Apr/2021 10:52:21] "GET /static/style.css HTTP/1.1" 200 -
10.249.78.110 - - [06/Apr/2021 10:52:23] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
10.249.78.110 - - [06/Apr/2021 10:52:23] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
10.249.78.110 - - [06/Apr/2021 10:52:23] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
10.249.78.110 - - [06/Apr/2021 10:52:24] "GET /static/favicon.ico HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 14:29:51] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 14:41:18] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 14:41:18] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 14:41:18] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 14:41:18] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 14:41:18] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 16:31:25] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 16:31:25] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 16:31:25] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 16:31:25] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 16:31:25] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 16:58:10] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 16:58:10] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 16:58:10] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 16:58:10] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 16:58:10] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 17:02:53] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 17:02:53] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 17:02:53] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 17:02:53] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 17:02:53] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 17:09:52] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 17:09:52] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 17:09:52] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 17:09:52] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 17:09:52] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 19:54:05] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [06/Apr/2021 19:54:05] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 19:54:06] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 19:54:06] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [06/Apr/2021 19:54:06] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [07/Apr/2021 09:37:42] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [07/Apr/2021 09:42:52] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [07/Apr/2021 09:42:53] "GET /static/pics/weather_forecast/%E9%9B%B7%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [07/Apr/2021 09:42:53] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.1 - - [07/Apr/2021 09:42:53] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [07/Apr/2021 09:42:53] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
192.168.31.1 - - [07/Apr/2021 10:38:21] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [07/Apr/2021 10:38:21] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.1 - - [07/Apr/2021 10:38:21] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [07/Apr/2021 10:38:21] "GET /static/pics/weather_forecast/%E9%9B%B7%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [07/Apr/2021 10:38:21] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
10.249.78.110 - - [08/Apr/2021 19:15:57] "GET /index HTTP/1.1" 200 -
10.249.78.110 - - [08/Apr/2021 19:15:58] "GET /static/pics/weather_forecast/%E9%9B%B7%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
10.249.78.110 - - [08/Apr/2021 19:15:58] "GET /static/style.css HTTP/1.1" 304 -
10.249.78.110 - - [08/Apr/2021 19:15:58] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 304 -
10.249.78.110 - - [08/Apr/2021 19:15:58] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 304 -
10.249.78.110 - - [08/Apr/2021 19:15:58] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [08/Apr/2021 20:01:20] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [08/Apr/2021 20:01:20] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.1 - - [08/Apr/2021 20:01:20] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [08/Apr/2021 20:01:20] "GET /static/pics/weather_forecast/%E9%9B%B7%E9%98%B5%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [08/Apr/2021 20:01:20] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [09/Apr/2021 10:34:34] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [09/Apr/2021 10:34:34] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [09/Apr/2021 10:34:34] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [09/Apr/2021 10:34:34] "GET /static/pics/weather_forecast/%E5%B0%8F%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [09/Apr/2021 16:14:48] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [09/Apr/2021 16:14:48] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.1 - - [09/Apr/2021 16:14:49] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [09/Apr/2021 16:14:49] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
10.250.204.62 - - [09/Apr/2021 23:22:48] "GET /index HTTP/1.1" 200 -
10.249.78.110 - - [10/Apr/2021 15:40:01] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [11/Apr/2021 13:15:15] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [11/Apr/2021 13:32:08] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [11/Apr/2021 13:32:09] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [11/Apr/2021 13:32:09] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [11/Apr/2021 13:32:09] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [12/Apr/2021 14:30:18] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [12/Apr/2021 14:30:18] "GET /static/style.css HTTP/1.1" 304 -
192.168.31.1 - - [12/Apr/2021 14:30:18] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 304 -
192.168.31.1 - - [12/Apr/2021 14:30:18] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 304 -
192.168.31.1 - - [13/Apr/2021 10:20:21] "GET /index HTTP/1.1" 200 -
192.168.31.1 - - [13/Apr/2021 10:20:22] "GET /static/pics/weather_forecast/%E9%98%B4.png HTTP/1.1" 200 -
192.168.31.1 - - [13/Apr/2021 10:20:22] "GET /static/style.css HTTP/1.1" 200 -
192.168.31.1 - - [13/Apr/2021 10:20:22] "GET /static/pics/weather_forecast/%E9%98%B5%E9%9B%A8.png HTTP/1.1" 200 -
192.168.31.1 - - [13/Apr/2021 10:20:22] "GET /static/pics/weather_forecast/%E5%A4%9A%E4%BA%91.png HTTP/1.1" 200 -
 * Tip: There are .env files present. Do "pip install python-dotenv" to use them.
 * Serving Flask app "flaskr" (lazy loading)
 * Environment: development
 * Debug mode: on
 * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Tip: There are .env files present. Do "pip install python-dotenv" to use them.
 * Debugger is active!
 * Debugger PIN: 138-361-213
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/usr/local/lib/python3.7/dist-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
10.249.144.233 - - [18/May/2021 16:38:20] "GET /index HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/index.py", line 11, in create
    curr_city = getLocalCityName()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/crawler_utils.py", line 19, in getLocalCityName
    return getCityName(getIPAddress())
  File "/home/pi/EpsAvlc_index/flaskr/crawler/crawler_utils.py", line 8, in getIPAddress
    ip = requests.get('https://checkip.amazonaws.com').text.strip()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='checkip.amazonaws.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb3da4630>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
10.249.144.233 - - [18/May/2021 16:38:20] "GET /index?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:38:20] "GET /index?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:38:20] "GET /index?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:38:21] "GET /index?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:38:52] "GET /index HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/index.py", line 11, in create
    curr_city = getLocalCityName()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/crawler_utils.py", line 19, in getLocalCityName
    return getCityName(getIPAddress())
  File "/home/pi/EpsAvlc_index/flaskr/crawler/crawler_utils.py", line 8, in getIPAddress
    ip = requests.get('https://checkip.amazonaws.com').text.strip()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='checkip.amazonaws.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb3b846f0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
10.249.144.233 - - [18/May/2021 16:38:52] "GET /index?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:38:52] "GET /index?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:38:52] "GET /index?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:38:53] "GET /index?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:01] "GET / HTTP/1.1" 302 -
10.249.144.233 - - [18/May/2021 16:39:12] "GET /index HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/index.py", line 11, in create
    curr_city = getLocalCityName()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/crawler_utils.py", line 19, in getLocalCityName
    return getCityName(getIPAddress())
  File "/home/pi/EpsAvlc_index/flaskr/crawler/crawler_utils.py", line 8, in getIPAddress
    ip = requests.get('https://checkip.amazonaws.com').text.strip()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='checkip.amazonaws.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb3bd5730>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
10.249.144.233 - - [18/May/2021 16:39:12] "GET /index?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:12] "GET /index?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:12] "GET /index?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:12] "GET /index?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:13] "GET /onmyoji HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:13] "GET /static/style.css HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:14] "GET /static/pics/onmyoji/xuanshang.png HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:14] "GET /static/favicon.ico HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:16] "GET /fund HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/fund.py", line 11, in create
    fund_list = get_fund_dict()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/fund_crawler.py", line 15, in get_fund_dict
    web_data = requests.get(fund_site, headers=headers).json()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 317, in prepare
    self.prepare_headers(headers)
  File "/usr/local/lib/python3.7/dist-packages/requests/models.py", line 451, in prepare_headers
    check_header_validity(header)
  File "/usr/local/lib/python3.7/dist-packages/requests/utils.py", line 960, in check_header_validity
    "bytes, not %s" % (name, value, type(value)))
requests.exceptions.InvalidHeader: Value for header {Upgrade-Insecure-Requests: 1} must be of type str or bytes, not <class 'int'>
10.249.144.233 - - [18/May/2021 16:39:16] "GET /fund?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:16] "GET /fund?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:16] "GET /fund?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:16] "GET /fund?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:30] "GET /index HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2309, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2295, in wsgi_app
    response = self.handle_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1741, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/EpsAvlc_index/flaskr/blueprints/index.py", line 11, in create
    curr_city = getLocalCityName()
  File "/home/pi/EpsAvlc_index/flaskr/crawler/crawler_utils.py", line 19, in getLocalCityName
    return getCityName(getIPAddress())
  File "/home/pi/EpsAvlc_index/flaskr/crawler/crawler_utils.py", line 8, in getIPAddress
    ip = requests.get('https://checkip.amazonaws.com').text.strip()
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='checkip.amazonaws.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb2f38cf0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
10.249.144.233 - - [18/May/2021 16:39:30] "GET /index?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:31] "GET /index?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:31] "GET /index?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1" 200 -
10.249.144.233 - - [18/May/2021 16:39:31] "GET /index?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 -
